cli/cmd/env_detection.go

```diff
@@ -47,6 +48,15 @@ func detectKernel() error {
 	return nil
 }
 func detectBpfCap() error {
+	err := rlimit.RemoveMemlock()
```

Comment:
add detection of root permissions

Reply:
> please add detection of root permissions

Let us check BPF_CAP by [man 2 capget](https://man7.org/linux/man-pages/man2/capset.2.html).

If we want to check more capabilities, we can check them by this way, too.